powersimdata/network/constants/region/europe.py

```diff
@@ -0,0 +1,257 @@
+from itertools import combinations
+
+import pandas as pd
+
+from powersimdata.network.constants.model import model2interconnect
+from powersimdata.network.helpers import interconnect_to_name
+
+abv2country = {
+    "AL": "Albania",
+    "AT": "Austria",
+    "BA": "Bosnia And Herzegovina",
+    "BE": "Belgium",
+    "BG": "Bulgaria",
+    "CH": "Switzerland",
+    "CZ": "Czech Republic",
+    "DE": "Germany",
+    "DK": "Danemark",
+    "EE": "Estonia",
+    "ES": "Spain",
+    "FI": "Finland",
+    "FR": "France",
+    "GB": "Great Britain",
+    "GR": "Greece",
+    "HR": "Croatia",
+    "HU": "Hungary",
+    "IE": "Ireland",
+    "IT": "Italy",
+    "LT": "Lithuania",
+    "LU": "Luxembourg",
+    "LV": "Latvia",
+    "ME": "Montenegro",
+    "MK": "Macedonia",
+    "NL": "Netherlands",
+    "NO": "Norway",
+    "PL": "Poland",
+    "PT": "Portugal",
+    "RO": "Romania",
+    "RS": "Serbia",
+    "SE": "Sweden",
+    "SI": "Slovenia",
+    "SK": "Slovakia",
+}
+
+abv2timezone = {
+    "AL": "ETC/GMT-1",
+    "AT": "ETC/GMT-1",
+    "BA": "ETC/GMT-1",
+    "BE": "ETC/GMT-1",
+    "BG": "ETC/GMT-2",
+    "CH": "ETC/GMT-1",
+    "CZ": "ETC/GMT-1",
+    "DE": "ETC/GMT-1",
+    "DK": "ETC/GMT-1",
+    "EE": "ETC/GMT-2",
+    "ES": "ETC/GMT-1",
+    "FI": "ETC/GMT-2",
+    "FR": "ETC/GMT-1",
+    "GB": "ETC/GMT",
+    "GR": "ETC/GMT-2",
+    "HR": "ETC/GMT-1",
+    "HU": "ETC/GMT-1",
+    "IE": "ETC/GMT",
+    "IT": "ETC/GMT-1",
+    "LT": "ETC/GMT-2",
+    "LU": "ETC/GMT-1",
+    "LV": "ETC/GMT-2",
+    "ME": "ETC/GMT-1",
+    "MK": "ETC/GMT-1",
+    "NL": "ETC/GMT-1",
+    "NO": "ETC/GMT-1",
+    "PL": "ETC/GMT-1",
+    "PT": "ETC/GMT",
+    "RO": "ETC/GMT-2",
+    "RS": "ETC/GMT-1",
+    "SE": "ETC/GMT-1",
+    "SI": "ETC/GMT-1",
+    "SK": "ETC/GMT-1",
+}
+
+
+interconnect2abv = {
+    "ContinentalEurope": {
+        "AL",
+        "AT",
+        "BA",
+        "BE",
+        "BG",
+        "CH",
+        "CZ",
+        "DE",
+        "DK",
+        "ES",
+        "FR",
+        "GR",
+        "HR",
+        "HU",
+        "IT",
+        "LU",
+        "ME",
+        "MK",
+        "NL",
+        "PL",
+        "PT",
+        "RO",
+        "RS",
+        "SI",
+        "SK",
+    },
+    "Nordic": {"FI", "NO", "SE"},
+    "GreatBritain": {"GB"},
+    "Ireland": {"IE"},
+    "Baltic": {"EE", "LT", "LV"},
+}
```

Comment:
Could we combine all of this information into a CSV, indexed by the country abbreviation, with columns for the long name, the time zone, and the interconnect? We would need to do a bit of reverse-mapping to get `interconnect2abv` back but I don't think it would be too bad.

Reply:
We can do that. We would have a *zone.csv* in the *powersimdata/network/europe_tub/data* folder?